Protein/examples/0.9-DRAFT/STRING_9606.ENSP00000379258.json

```diff
@@ -8,23 +8,26 @@
       "@base": "http://schema.org"
     }
   ],
-  "@type": "Dataset",
+  "@type": "bio:DataRecord",
   "@id": "https://string-db.org/network/9606.ENSP00000379258",
+  "http://purl.org/dc/terms/conformsTo": "https://bioschemas.org/specifications/DataRecord/0.2-DRAFT",
+  "includedInDataset": "https://string-db.org/#string.v11",
   "mainEntity": {
-    "@type": [
-      "bio:Protein"
-    ],
+    "@type": "bio:Protein",
+    "@id": "https://string-db.org/network/9606.ENSP00000379258",
+    "http://purl.org/dc/terms/conformsTo": "https://bioschemas.org/specifications/Protein/0.9-DRAFT",
```

Comment:
The currentProtein profile version is https://bioschemas.org/profiles/Protein/0.11-RELEASE/ should it not be better to update the example to the latest release rather than using an old DRAFT profile?